cmd/porter/bundle.go

```diff
@@ -31,14 +31,23 @@ func buildBundleCommands(p *porter.Porter) *cobra.Command {
 }
 
 func buildBundleCreateCommand(p *porter.Porter) *cobra.Command {
-	return &cobra.Command{
-		Use:   "create",
+	cmd := &cobra.Command{
+		Use:   "create [bundle-name]",
 		Short: "Create a bundle",
-		Long:  "Create a bundle. This generates a porter bundle in the current directory.",
+		Long:  "Create a bundle. This generates a porter bundle with the directory with the specified name or in the root directory if no name is provided.",
+		Args:  cobra.MaximumNArgs(1), // Expect at most one argument for the bundle name
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return p.Create()
+			bundleName := "" // Default value
```

Comment:
```suggestion
                        // By default we create the bundle in the current directory
			bundleName := ""
```

Minor comment nit

Reply:
By the way it is hard to do suggestions for comments in github because it won't let me use tabs :( so it may look a little weird

Reply:
Yeah, but I got it anyways! would fix them up.